<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeData</class>
 <widget class="QWidget" name="ChangeData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>1055</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>290</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Изменить имя секции</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Старое имя секции</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxOldNameSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новое имя секции</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewNameSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonChangeName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>600</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Изменить комментарий в секции</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя секции</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxNameSectionForChangeComment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>220</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новый комментарий в секции</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewComment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>580</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChangeComment">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>290</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Изменить ключ секции</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя секции</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxNameSectionForChangeKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Старое значение ключа</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOldNameKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChangeKey">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Новое значение ключа</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNewNameKey">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>119</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>290</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Изменить значение в ключе</string>
   </property>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя секции</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxNameSectionForChangeValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя ключа</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNameKeyForChangeValue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOldvalue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Старое значение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Новое значение</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewValue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChangeValue">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>290</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Добавить новое значение</string>
   </property>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя секции</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxNameSectionForAddValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя ключа</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewValue_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>119</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя значения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddNewValue">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>600</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Добавить комментарий в секцию</string>
   </property>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя секции</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxNameSectionForAddComment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Комментарий</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddNewComment">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNewComment_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>580</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменённая секция</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>580</width>
     <height>370</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
